content/docs/other-guides/accessing-uis.md

```diff
@@ -9,50 +9,45 @@ instructions on how to connect to them.
 
 ## Accessing Kubeflow web UIs
 
-Kubeflow comes with a number of web UIs, including:
+The Kubeflow web UIs include the following:
 
-* Central UI for navigation
-* Jupyter notebooks
-* TFJob Dashboard
-* Katib Dashboard
-* Pipelines Dashboard
-* Artifact Store Dashboard
+* A central **Kubeflow** UI for navigation between the Kubeflow applications.
+* **Pipelines** for a Kubeflow Pipelines dashboard
+* **Notebook Servers** for Jupyter notebooks.
+* **Katib** for hyperparameter tuning.
+* **Artifact Store** for tracking of artifact metadata.
+* **tf-operator** for a TFJob dashboard.
 
-To make it easy to connect to these UIs Kubeflow provides a left hand navigation
-bar for navigating between the different applications.
+Instructions below indicate how to connect to the Kubeflow central UI. From
+there you can navigate to the different services using the left hand navigation
+bar. 
 
-Instructions below indicate how to connect to the Kubeflow landing page. From
-there you can easily navigate to the different services using the left hand navigation
-bar. The landing page looks like this:
+The central UI dashboard looks like this:
 
 <img src="/docs/images/central-ui.png"
-  alt="Kubeflow UI"
+  alt="Kubeflow central UI"
   class="mt-3 mb-3 border border-info rounded">
 
+## URL pattern with Google Cloud Platform (GCP)
 
-## Google Cloud Platform (Kubernetes Engine)
-
-If you followed the guide to [deploying Kubeflow on Google Cloud Platform
-(GCP)](/docs/gke/deploy/), Kubeflow
-is deployed with Cloud Identity-Aware Proxy (Cloud IAP) or basic authentication,
-and the Kubeflow landing page is accessible at a URL of the following pattern:
+If you followed the guide to [deploying Kubeflow on GCP](/docs/gke/deploy/), 
+the Kubeflow central UI is accessible at a URL of the following pattern:
 
 ```
-https://<name>.endpoints.<project>.cloud.goog/
+https://<application-name>.endpoints.<project-ID>.cloud.goog/
```

Comment:
Nit: Since the variables in the URL pattern are presented in lowercase, should "project-ID" be "project-id"?

Reply:
It should. Done.